plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/exec/output/DBCOutputSeverity.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.model.exec.output;
+
+import org.jkiss.dbeaver.model.DBPNamedObject;
+
+public interface DBCOutputSeverity extends DBPNamedObject {
```

Comment:
> Missing a Javadoc comment.

I agree here with the code style because you have no methods, and it is more complicated to understand - what for this interface?